pkgs/dartpad_ui/lib/main.dart

```diff
@@ -1381,19 +1391,18 @@ class GeminiMenu extends StatelessWidget {
         ...[
           MenuItemButton(
             leadingIcon: image,
-            onPressed: generateNewCode,
-            child: const Padding(
-              padding: EdgeInsets.only(right: 32),
-              child: Text('Generate code'),
-            ),
+            onPressed: generateNewDartCode,
+            child: menu('Dart Snippet'),
+          ),
+          MenuItemButton(
+            leadingIcon: image,
+            onPressed: generateNewFlutterCode,
+            child: menu('Flutter Snippet'),
           ),
           MenuItemButton(
             leadingIcon: image,
             onPressed: updateExistingCode,
-            child: const Padding(
-              padding: EdgeInsets.only(right: 32),
-              child: Text('Update code'),
-            ),
+            child: menu('Update code'),
```

Comment:
Why can't I see this option in https://genui-dartpad.web.app ?

Reply:
https://dartpad-polina--dartpad-polina-ohhq8wl3.web.app/